public/js/src/bridge/bridge_lb_status.tag

```diff
@@ -0,0 +1,69 @@
+<bridge_lb_status>
```

Comment:
Let me test to confirm. But my hope was that the interval would continue so that someone could check on other things and then come back and see if the LB were ready.